CONTRIBUTING.md

```diff
@@ -0,0 +1,233 @@
+6. Launch the Live Server:
+- In the opened HTML file, you should see a "Go Live" button in the bottom-right corner of the VS Code window. Click this button to launch the live server.
+- This will open your documentation in a new browser tab at a local address (typically `http://127.0.0.1:5500`)
+7. Make and Test Changes:
+-With the Live Server running, you can make changes to the documentation files using VS Code, and the changes will be automatically reflected in your browser. Test your changes thoroughly.
+8. Stop the Live Server:
+- When you're done testing or making changes, you can stop the Live Server by clicking the "Exit" button in the VS Code status bar.
+
+## Adding Translations to Documentation
+Adding translations to our project's documentation helps make it more accessible to a wider audience. If you're interested in contributing translations for our documentation, follow these steps
+Steps to Add Translations
+1. Identify Target Languages:
+- Determine which languages you want to add translations for. Make sure these languages are relevant to the project's user base.
+2. Create Translation Files:
+- Inside the documentation directory, create a new subdirectory for each language you plan to support. Use language codes (e.g., "en" for English, "fr" for French) as directory names.
+
+```bash
+- docs/
+  - en/         (English)
+  - fr/         (French)
+  - es/         (Spanish)
+```
+3. Translate Content:
+- For each language directory, create translated versions of the documentation files. Typically, you will translate Markdown files, but consider other formats as needed.
+- Maintain the same file structure and filenames as in the original documentation, but with translated content.
+4. Update Links:
+- In the translated files, ensure that any internal links (e.g., links to other sections or pages within the documentation) are updated to point to the corresponding translated content.
+5. Add Language Selector:
+- Consider adding a language selector to the documentation, allowing users to switch between different language versions. You can do this by modifying the navigation or header of your documentation.
+6. Testing and Validation:
+- Test the translated documentation to ensure accuracy and readability. Make sure that all links work correctly and that the content is culturally appropriate.
+7. Submit Contributions:
+- If you haven't already, submit your translations as a pull request. Ensure that you provide clear information about the languages you've translated and any specific details related to your contributions.
+8. Collaborate and Review:
+- Collaborate with other contributors and reviewers to ensure the quality of translations. Be open to feedback and suggestions for improvement.
+
+Best Practices for Translations
+- Maintain consistency in terminology and style throughout the translated documentation.
+- Consider using translation tools or services to assist with the translation process.
```

Comment:
[proofreading] It helps mentioning specific tools so that developers who want to translate documentation can see how it's done.
```suggestion
- Consider using translation tools or services like [Crowdin](https://crowdin.com/) and [Google Translate](https://translate.google.com/about/) to assist with the translation process.
```